src/metabase/upload.clj

```diff
@@ -68,6 +69,29 @@
   (let [column-limit (some-> driver driver/column-name-length-limit)]
     (min-safe column-limit (max-bytes :model/Field :name))))
 
+(defn truncate-utf8-bytes
```

Comment:
This really sucks, I'm surprised there is not something built in. Maybe need to look harder?

Reply:
Pushed this to a later PR